python/tank/pipelineconfig.py

```diff
@@ -178,27 +235,23 @@ def get_all_os_paths(self):
     def get_name(self):
         """
         Returns the name of this PC.
-        May connect to Shotgun to retrieve this.
         """
-        if self._pc_name is None:
-            # try to get it from the cache file
-            data = pipelineconfig_utils.get_metadata(self._pc_root)
-            self._pc_name = data.get("pc_name")
-
-
-            if self._pc_name is None:
-                # not in metadata file on disk. Fall back on SG lookup
-                self._load_metadata_from_sg()
-
         return self._pc_name
 
     def is_auto_path(self):
         """
         Returns true if this config was set up with auto path mode.
-        This method will connect to shotgun in order to determine the auto path status.
-        
+        This method will connect to shotgun in order to determine the 
+        auto path status.
+
+        January 2016:
+        DEPRECATED - DO NOT USE! At some stage this wil be removed.
```

Comment:
Deprecated calls sounds great! Part of further logging work I guess! Re deprecation - I am guessing we won't take this out for at least a year. It's a warning saying 'don't write *new* code that calls this.' So I think we are good for the time being. Backwards compatbility is key!